src/main/sensors/battery.c

```diff
@@ -116,7 +116,7 @@ PG_RESET_TEMPLATE(batteryConfig_t, batteryConfig,
     .vbatfullcellvoltage = 41,
 
     .vbatLpfPeriod = 35,
-    .ibatLpfPeriod = 10,
+    .ibatLpfPeriod = 40,
```

Comment:
please check on https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history and learn how to interactively rebase so the change is not part of any commit